apps/ledger-live-mobile/src/screens/Swap/Form/Connect.tsx

```diff
@@ -22,7 +22,7 @@ export function Connect({
   provider?: string;
 }) {
   const [device, setDevice] = useState<Device>();
-  const [result] = useState();
+  const [result] = useState(); // WTF
```

Comment:
Can you help me figure out what's going on here @sarneijim ? I've not seen a useState get used like this before

Reply:
Second param is optional, as `setResult` is not going to be used, you do not need to set up. (Similar what we saw this morning second argument is not required).
```
const [result, functionToSetNotUsed] = useState(); 
===
const [result] = useState();

Reply:
`setResult` is passed in as a function to the component and used when hiding the modal. Maybe it's fine, just looks a bit funny